vendor/github.com/vmware/photon-controller-go-sdk/SSPI/sspi.go

```diff
@@ -0,0 +1,276 @@
+// +build windows
+
+package SSPI
+
+import (
+	"fmt"
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	secur32_dll           = syscall.NewLazyDLL("secur32.dll")
+	initSecurityInterface = secur32_dll.NewProc("InitSecurityInterfaceW")
```

Comment:
Hi Jordan. Thanks for your review. I have talked to Photon team who's maintaining this go sdk. We think this usage should be safe for 1) this will only be built on Windows and 2) this refers to standard Windows stuff. Please let us know if you still have concern about this part of code.